src/commands/slugs/download.ts

```diff
@@ -0,0 +1,54 @@
+import {Command, flags} from '@heroku-cli/command'
+import * as Heroku from '@heroku-cli/schema'
+import {Args, ux} from '@oclif/core'
+import {execSync} from 'node:child_process'
+
+import {download} from '../../lib/download'
+
+export default class SlugsDownload extends Command {
+  static args = {
+    slug: Args.string({description: 'slug ID or name of slug'}),
+  }
+
+  static description = 'downloads a slug to <APP_NAME>/slug.tar.gz and then extracts it'
+
+  static examples = [
+    '$ heroku slugs:download --app myapp v2',
```

Comment:
```suggestion
    '$ heroku slugs:download --app example-app v2',
```

Reply:
I guess `v2` is the name of the slug in this example?

Reply:
yep!